src/main/java/appeng/util/AlphanumComparator.java

```diff
@@ -0,0 +1,108 @@
+package appeng.util;
+// spotless:off
+/*
```

Comment:
Please include a note about the library this code is from. Otherwise license statement 'this library' looks confusing.